packages/react/src/components/input/input.tsx

```diff
@@ -44,4 +46,12 @@ export const Input = withRef(
 );
 Input.displayName = 'Input';
 
-export type InputProps = BaseProps & JSX.IntrinsicElements['input'];
+export type InputProps = BaseProps &
+  Omit<InputElementProps, 'children'> & {
+    /** @deprecated Use `label` prop instead */
+    children?: InputElementProps['children'];
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    label?: any;
```

Comment:
```suggestion
    label?: React.ReactNode;
```

Reply:
Hmm, you think so?

I was looking at what usually `children` is, which essentially was what used previously, and seems like it's just `any`?

Reply:
Yea pretty sure
```ts
interface DOMAttributes<T> {
        children?: ReactNode;
```

Reply:
There are other instances around other files, I'd recommend importing `{ ReactNode } from 'react'` and updating them all.

Reply:
Yep, done that locally.